panel_view.py

```diff
@@ -0,0 +1,60 @@
+import sublime
+import sublime_plugin
+
+from .lint import persist
+from .lint import util
+from .lint import events
+from .panel import panel
+
+
+State = {
```

Comment:
I wonder if the state could be shared among the multiple implementations. Maintaining the same state twice does seem redundant and where most of the work ends up in. It would require a dedicated set of event hooks that update the state as well as an event mechanic notifying potential listeners of a state update. I'm not up to date on whether the internal event mechanic still exists.

I think traditionally the persist module was used for storing things like this a lot.

PS: also, can we please not use TitleCase for non-classes?